### PR TITLE
[DOCS] Adds force query parameter to the stop API parms

### PR DESCRIPTION
**Pull request by szabosteve**
This PR adds `force` query parameter to the parameter pool of the stop transform API.

Related issue: https://github.com/elastic/ml-team/issues/187#issuecomment-529352714

### CODE REVIEW

File: docs/reference/data-frames/apis/stop-transform.asciidoc [hunk 1]
```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed job back to 
+  started. It is required to do in order to delete or restart the job (after
```
Comment (lcawl) [inline docs/reference/data-frames/apis/stop-transform.asciidoc]: This seems unexpected.  Does "stopping" the transform forcefully really put it into "started" state?

File: docs/reference/data-frames/apis/stop-transform.asciidoc [hunk 2]
```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed job back to
```
Comment (lcawl) [inline docs/reference/data-frames/apis/stop-transform.asciidoc]:
```suggestion
  (Optional, boolean) If `true`, resets the state of a failed {transform} back to 
```

File: docs/reference/data-frames/apis/stop-transform.asciidoc (hunk 1 above)
Comment (lcawl) [reply docs/reference/data-frames/apis/stop-transform.asciidoc]: I would have expected the description for this parameter to be similar to the force parameter on the close job API: https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-close-job.html

Comment (benwtrent) [reply docs/reference/data-frames/apis/stop-transform.asciidoc]: @lcawl is correct

File: docs/reference/data-frames/apis/stop-transform.asciidoc [hunk 3]
```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed {transform} back to
```
Comment (szabosteve) [inline docs/reference/data-frames/apis/stop-transform.asciidoc]:
```suggestion
  (Optional, boolean) If `true`, resets the state of a failed {transform} to 
```

File: docs/reference/data-frames/apis/stop-transform.asciidoc [hunk 4]
```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed {transform} to 
+  started. It is required to do in order to delete or restart the job (after
```
Comment (szabosteve) [inline docs/reference/data-frames/apis/stop-transform.asciidoc]:
```suggestion
  stopped. It is required to do in order to delete or restart the job (after 
```

File: docs/reference/data-frames/apis/stop-transform.asciidoc (hunk 1 above)
Comment (szabosteve) [reply docs/reference/data-frames/apis/stop-transform.asciidoc]: Sorry, my bad. I corrected it.

File: docs/reference/data-frames/apis/stop-transform.asciidoc [hunk 5]
```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed {transform} to
```
Comment (lcawl) [inline docs/reference/data-frames/apis/stop-transform.asciidoc]: I suggest changing the description of the "force" parameter as follows to be closer to what's in https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-close-job.html :

>  Use to close a failed {transform} or to forcefully close a {transform} that has not responded to its initial stop request.

**Comment by szabosteve** [comment]
I close this PR and open a new one because of the new transforms folder structure.

**Pull request closed without merging**